guides/common/modules/proc_installing-fapolicyd-on-server.adoc

```diff
@@ -0,0 +1,61 @@
+[id="installing-fapolicyd-on-server_{context}"]
+ifeval::["{context}" == "{project-context}"]
+= Installing fapolicyd on {ProjectServer}
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+= Installing fapolicyd on {SmartProxyServer}
+endif::[]
+
+ifeval::["{context}" == "{project-context}"]
+fapolicyd can either be installed together with {ProjectServer} or can be installed on an existing {ProjectServer}.
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+fapolicyd can either be installed together with {SmartProxyServer} or can be installed on an existing {SmartProxyServer}.
+endif::[]
+Use the following procedures to install fapolicyd on your {EL} host.
+
+.Prerequisite
+* Ensure your host has access to the base repositories of {EL}.
+
+.Procedure
+. Install fapolicyd:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+ifndef::foreman-deb[]
+# dnf install fapolicyd
+endif::[]
+ifdef::foreman-deb[]
+# {package-install-project} fapolicyd
+endif::[]
+----
++
+. Start fapolicyd service:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# systemctl enable --now fapolicyd
+----
+
+.Verification
+* Enter the following command to verify the fapolicyd service is running correctly:
++
+[options="nowrap" subs="+quotes"]
+----
+# systemctl status fapolicyd
+----
+
+In case of new {Project} or {SmartProxy} installation, follow the standard installation procedures after installing and enabling fapolicyd on your {EL} host.
+
+[NOTE]
+====
+ifeval::["{context}" == "{project-context}"]
+If you are installing fapolicyd along with the new {ProjectServer}, the installation process will detect the fapolicyd in your {EL} host and deploy the {Project} rules automatically.
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+If you are installing fapolicyd along with the new {SmartProxyServer}, the installation process will detect the fapolicyd in your {EL} host and deploy the {SmartProxy} rules automatically.
+endif::[]
+====
```

Comment:
What would you think about moving this to the procedure introduction -- perhaps in the form of a deicision tree? E.g. "To install fapolicyd when installing a new server" and "To install fapolicyd on an existing server". My concern is that included here at the bottom of the module, the information is not visible enough.